src/utils/framediff_utils.py

```diff
@@ -0,0 +1,93 @@
+import cv2
+import numpy as np
+
+"""
+
+Utility functions to calculate inter-frame difference
+
+"""
+
+class DistanceMetrics:
+	def convert_to_grayscale(self, frame):
+		return cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
+		#return frame
+
+	def absolute_difference(self, curr_frame, prev_frame):
+		curr_frame_grayscale = self.convert_to_grayscale(curr_frame)
+		prev_frame_grayscale = self.convert_to_grayscale(prev_frame)
+		frame_diff = cv2.absdiff(curr_frame_grayscale, prev_frame_grayscale)
+		return frame_diff.sum()
+
+	def mse_difference(self, curr_frame, prev_frame):
+		curr_frame_grayscale = self.convert_to_grayscale(curr_frame)
+		prev_frame_grayscale = self.convert_to_grayscale(prev_frame)
+		total_pixels = curr_frame_grayscale.shape[0]*curr_frame_grayscale.shape[1]
+		frame_diff = curr_frame_grayscale.astype('float') - prev_frame_grayscale
+		mse_diff = np.sum(frame_diff**2)/float(total_pixels)
+		return mse_diff
+
+# Function to blacken out background of frame
+def mask_background(frame):
```

Comment:
Can we just plug out all the hardcoded variables from this function and pass them as parameters. **kwargs is normally the way to do this in python. I mean the users can pass whatever parameters they want to control and rest can be defaults.